### ISSUE TITLE
Bad engine config

### ISSUE THREAD

**Issue by willbamford** [sogame/danger-plugin-toolbox#71]
Currently the `engine` configuration doesn't allow for Node 11 (https://github.com/sogame/danger-plugin-toolbox/blob/master/package.json#L7).

The following would be more future / now friendly:

```
"node": ">=10.13.0",
```

**Comment by sogame** [comment]
Hi @WebSeed,

Thanks for creating this issue. Ideally development (and therefore running tests) should always be done in the version defined in `engines`. Nvm makes it very easy to switch to a different version.

Anyways, I've no problem allowing newer versions of Node (as CI will use the defined one), so I've updated the configuration: https://github.com/sogame/danger-plugin-toolbox/pull/75

**Comment by steprescott** [comment]
Hi @sogame 

Using the latest release of danger [package.json](https://github.com/danger/danger-js/blob/master/package.json#L109) it has a dependency on `"@types/node": "^10.11.3"`. Your [package.json](https://github.com/sogame/danger-plugin-toolbox/blob/master/package.json#L7) requires `"node": "^12.14.1"`.

I am using Docker and when building the image I get the error below. With your plugin, which by the way will be very useful for my needs so thanks for the work, wanting a version higher than what danger requires how dose this match up?

Thanks for any help you can give.

```
$ node out/scripts/setup-plugins.js
winston:create-logger Define prototype method for "error"
winston:create-logger Define prototype method for "warn"
winston:create-logger Define prototype method for "info"
winston:create-logger Define prototype method for "http"
winston:create-logger Define prototype method for "verbose"
winston:create-logger Define prototype method for "debug"
winston:create-logger Define prototype method for "silly"
Installing: danger-plugin-jira-issue, danger-plugin-toolbox
-> : [1/5] Validating package.json...

-> : [2/5] Resolving packages...

-> : [3/5] Fetching packages...

-> : info fsevents@1.2.4: The platform "linux" is incompatible with this module.

-> : info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.

! -> : error danger-plugin-toolbox@1.17.1: The engine "node" is incompatible with this module. Expected version "^12.13.1". Got "10.17.0"

! -> : error Found incompatible module.

-> : info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.

child process exited with code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run setup:plugins' returned a non-zero code: 1
```

**Comment by sogame** [comment]
Hi @steprescott,

I've just published version `1.18.0` changing the Node requirement to 10.13 (Dubnium), I hope this solves the issues :-)

**Comment by steprescott** [comment]
This worked thanks. 👍
